src/metpy/io/gempak.py

```diff
@@ -1270,12 +1271,12 @@ def gdxarray(self, parameter=None, date_time=None, coordinate=None,
                             gvcord: [col_head.GLV1],
                             'x': self.x,
                             'y': self.y,
+                            'metpy_crs': CFProjection(self.crs.to_cf())
                         },
                         dims=['time', gvcord, 'y', 'x'],
                         name=var,
                         attrs={
-                            **self.crs.to_cf(),
-                            'grid_type': ftype,
+                            'gempak_grid_type': ftype,
```

Comment:
What's the reason for changing this `'grid_type'` attribute?

Reply:
It seemed liable to be confused with `grid_mapping`. Can definitely change it back though!

Reply:
*mumbles something about not breaking stuff*

Reply:
To be clear, we can leave it...but mark those words if someone complains. 😆